main/options.c

```diff
@@ -348,6 +348,8 @@ static optionDescription LongOptionDescription [] = {
 #endif
  {0,"  --output-format=ctags|etags|xref"},
  {0,"      Specify the output format. [ctags]"},
+ {0,"  --pattern-length-limit=N"},
+ {0,"      Cutoff tag entries after N characters. Disable by setting to 0. [96]"},
```

Comment:
> "      Cutoff tag entries after ..."

I think this should be "      Cutoff patterns of tag entries after ...".
Am I wrong?